graph/patterns/dictionary/index.md

```diff
@@ -147,13 +178,13 @@ PATCH https://graph.microsoft.com/v1.0/users/10/roles/author
 }
 ```
 
-### Deleting an entry from the dictionary
+#### Deleting an entry from the dictionary
 
 ```HTTP
 DELETE https://graph.microsoft.com/v1.0/users/10/roles/author
 ```
 
-## CDSL example
+### CDSL example
```

Comment:
"<!-- use this annotation to indicate you want the SDKs to generate additional request builders to update the dictionary atomtically -->"

No idea what this is about but it needs more explanation.